test/Header/SenderTest.php

```diff
@@ -190,4 +190,50 @@ public function invalidSenderEncodedDataProvider()
             'multiline' => array("<foo\r\n@\r\nbar>", $headerInvalidArgumentException, null),
         );
     }
+
+    /**
+     * @param string $headerString
+     * @param string $expectedName
+     * @param string $expectedEmail
+     * @param string $expectedException
+     *
+     * @dataProvider headerLinesProvider
+     */
+    public function testFromString($headerString, $expectedName, $expectedEmail, $expectedException = null)
+    {
+        if ($expectedException) {
+            $this->setExpectedException($expectedException);
+        }
+
+        $header = Header\Sender::fromString($headerString);
+
+
+        if ($expectedException) {
+            return;
+        }
+
+        $this->assertSame($expectedName, $header->getAddress()->getName());
+        $this->assertSame($expectedEmail, $header->getAddress()->getEmail());
+    }
+
+    public function headerLinesProvider()
+    {
+        $mailInvalidArgumentException = 'Zend\Mail\Exception\InvalidArgumentException';
+        $headerInvalidArgumentException = 'Zend\Mail\Header\Exception\InvalidArgumentException';
+
+        return array(
```

Comment:
now is mandatory use short array syntax. Next push will be checked against this new guidelines.